app/views/metrics/_metric_header.html.erb

```diff
@@ -1,4 +1,12 @@
-<h3><%= title %></h3>
-<div class="aggregated-value">
-  <%= value %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "components/info-metric", {
+        name: title,
+        figure: number_with_delimiter(value, :delimiter => ','),
```

Comment:
What about using 1.9 hash syntax here?

Reply:
@peteglondon can you address Erin's comment please and then we can merge. Not sure why the linter didn't spot that. Thank @erino

Reply:
Rubocop doesn't check erb, so there maybe other style issues.

Reply:
Ok I didn't spot that, done now.